library/build.gradle

```diff
@@ -10,6 +10,7 @@ android {
     defaultConfig {
         minSdkVersion minSdkVersionDeclared
         targetSdkVersion compileSdkVersionDeclared
+        testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
```

Comment:
Yes because the addition of instrumentation tests to the library

Reply:
Oh I didn't see that. Maybe we should move that test to the sample module. Because that's where we put Barista instrumentation tests.

That way we only need to run one set of Instrumentation tests, instead of two.

Reply:
I don't agree because the test is for an internal library class that should not be visible for the sample or any library's consumer so, why should the sample know anything about this class?

Reply:
`BitmapComparator` is public, isn't it? I mean, it's not internal at all...

Reply:
You are right @rocboronat, it should be internal and we will make it once this issue is solved: #124